packages/teleport-component-generator-vue/__tests__/integration/component-with-conditional-root-string-child-node.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://docs.teleporthq.io/uidl-schema/v1/component.json",
+    "name": "ComponentWithConditionalRootNode",
```

Comment:
Same as in the previous .json file. This file can be removed and its content should generated with uidl-builders functions